pkg/services/references_test.go

```diff
@@ -256,6 +256,12 @@ func TestReferences(t *testing.T) {
 			},
 		},
 	}
+	context := &utils.LsContext{
+		Api: utils.ApiContext{
+			Token:   "XXXXXXXXXXXX",
+			HostUrl: "https://circleci.com",
+		},
+	}
```

Comment:
instead of re-creating this object everywhere we won't use it, can we have a util function that create a dummy one ? that will avoid copy paste in the code and will be easier to change if we do a refactoring of the context in the future